pages/Transfer.qml

```diff
@@ -370,8 +371,9 @@ Rectangle {
               CheckBox {
                   id: paymentIdCheckbox
                   border: false
-                  checkedIcon: "qrc:///images/plus-in-circle-medium-white.png"
-                  uncheckedIcon: "qrc:///images/plus-in-circle-medium-white.png"
+                    checkedIcon: FontAwesome.minusCircle
```

Comment:
not indented

Reply:
Actually is the correct indentation. The lines above and below are not indented properly.

Reply:
yep, half of the Transfer page is all wrong with indentation. If you prefer to use right indentation with new code also ok

Reply:
> If you prefer to use right indentation with new code also ok

Yeah, kinda tired of manually adjusting the spaces